proto/beacon/rpc/v1/services.proto

```diff
@@ -81,14 +81,15 @@ message AttestResponse {
 
 message ValidatorPerformanceRequest {
   uint64 slot = 1;
-  bytes public_key = 2;
+  repeated bytes public_keys = 2;
 }
 
 message ValidatorPerformanceResponse {
-  uint64 balance = 1;
+  repeated uint64 balances = 1;
   uint64 total_validators = 2;
   uint64 total_active_validators = 3;
-  float average_active_validator_balance = 4;
+  repeated bytes missing_validators = 4;
```

Comment:
Why do we need a separate field for `missing_validators`, that's the same thing as balance equals to 0. Both can be used for the same context. I can't think of a use case where `missing_validators` is used over balance > 0

Reply:
discussed offline, this is good